wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/wsplugins/model/PluginMeta.java

```diff
@@ -22,6 +23,15 @@
   private String version = null;
   private String title = null;
   private String description = null;
+  private String category = null;
+  private String publisher = null;
+  private String repository = null;
+  private List<String> tags = null;
+  private String mediaImage = null;
+  private String mediaVideo = null;
+  private String firstPublicationDate = null;
```

Comment:
@slemeur is it OK that we have an inconsistent style of fields names in the YAML? 
`media-video` with dash
`firstPublicationDate` with camel case